app/views/users/show.html.haml

```diff
@@ -0,0 +1,28 @@
+%h1 Users
+
+%table.table-striped.table-bordered
+  %tr
+    %th Fullname
+    %th Actions
+  - @users.each do |user|
+    %tr
+      %td= user.fullname
+      %td
+        = link_to 'Trim', trim_user_path(user), :id => "#{user.id}", :class => 'trim', :data => {:url => '/trim'}
+        = link_to 'Massage and Trim', massage_and_trim_user_path(user), :id => "#{user.id}", :class=> 'massage-trim', :data => {:url => '/massage_and_trim'}
+        = link_to 'Shave', shave_user_path(user), :id => "#{user.id}", :class => "shave", :data => {:url => '/shave'}
+        = button_to "Reset", reset_user_path(user), :method => :put
+
+%h2 Sign up!
```

Comment:
Typically, this is what /new is for.

Please read: http://www.sitepoint.com/building-your-first-rails-application-views-and-controllers/
there is a great section there on RESTful controllers.